osgi/src/main/resources/resources/components/connections/DummyConnection.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```

Comment:
Why the duplication from core? Classloader issues? And if we are duplicating, should we do it for all? Are we supporting all these connections?